server/routes/handlers/outcomes/getCasesToResultHandler.js

```diff
@@ -1,20 +1,21 @@
 const getOutcomeTypesListFilters = require('../../../utils/getOutcomeTypesListFilters')
+const flagFilters = require('../../../utils/flagFilters')
 const getCasesToResultHandler = caseService => async (req, res) => {
   const {
     params: { courtCode, title, sorts, state },
     params
   } = req
 
-  const filters = getOutcomeTypesListFilters(req.query)
-
-  const filtersApplied = filters.map(filterObj => filterObj.items.filter(item => item.checked).length).pop()
-
-  const response = await caseService.getOutcomesList(courtCode, filters, sorts, state)
+  const response = await caseService.getOutcomesList(courtCode, req.query, sorts, state)
```

Comment:
Again the same with the query.... I feel like I'm being overly picky though so you can ignore...

Reply:
done